integrations/symfony-bundle.rst

```diff
@@ -288,6 +288,11 @@ You can configure a client with authentication. Valid authentication types are
                 factory: 'httplug.factory.guzzle6'
                 plugins: ['httplug.plugin.authentication.my_wsse']
 
+.. warning::
+
+    Using query parameters for authentication is :doc:`not safe </message/authentication#query-params>`.
```

Comment:
the most robust way to cross reference to an anchor is to define the target like `.. _ShortAnchor:` right above the heading we want to link to, and then use `` :ref:`ShortAnchor` ``

Reply:
edited the example for a custom text of the link

Reply:
> the most robust way to cross reference to an anchor is to define the target like `.. _ShortAnchor:`

I'm not sure to follow you. Could you please bring me a existing case?

Reply:
https://raw.githubusercontent.com/php-http/documentation/master/clients/curl-client.rst 
```
The cURL client needs a :ref:`message factory <message-factory>` and a
```

points to https://raw.githubusercontent.com/php-http/documentation/master/message/message-factory.rst
```
.. _message-factory:
```

(not a very good use case, as that could also just link to that doc file - the main point of anchors is to deep link into a page... but a working example)

Reply:
I'm sorry but I still not see how this will help me with `#query-params` anchor... ^^"